### DISCUSSION TITLE
Proposal: use replays made by players to substitute Auto replays when possible

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#23168] [Ideas] [answered]
Basically, when a player uses the Auto mod, they are basically wanting to see a perfect score on the map with a specified mod combo. Using player relays when possible grants the following 2 benefits:

1. Player gets to see a more "natural" perfect score than what auto does, because score is being made by an actual human, therefore providing a more intuitive and human experience to the spectator.
2. The player who has set the perfect score would get more recognition on their score in the form of views, giving a much better incentive for players to compete for 1st place scores, therefore proving an extra competitive ground for players.

Of course, a player score won't be available at all times, so the game should still fall back to actual auto if:

- Player is offline
- There isn't an available perfect score for the specified mod combo

In the case a player strictly wants to see an auto replay, a mod setting can be added that forces an auto replay to load.

Thoughts?

**Comment by Rekunan** [comment, marked as answer]
This seems unnecessary because if people are looking for the 2 benefits they could just search for an SS score, especially if they use accuracy sort.
